web/pageComponents/shared/IframeCarousel/IframeCarousel.tsx

```diff
@@ -25,8 +26,16 @@ const Figure = styled.figure`
   width: 100%;
 `
 const ItemContainer = styled.div`
+  min-width: 45%;
```

Comment:
with should aim to be mobile first, so min-width should be 100%, and then use @media(**min**-width: ) etcs..